conda/environment-unpinned.yml

```diff
@@ -14,9 +14,10 @@ dependencies:
   - cartopy
   - cf_xarray
   - dask
+  - datashader
```

Comment:
required for `hvplot.image(rasterize=True)`, but brings in numba which currently requires <py310